zkevm-circuits/src/mpt_circuit/rlp_gadgets.rs

```diff
@@ -37,12 +40,26 @@ pub(crate) struct RLPListWitness {
 
 impl<F: Field> RLPListGadget<F> {
     pub(crate) fn construct(cb: &mut ConstraintBuilder<F>, bytes: &[Expression<F>]) -> Self {
-        // TODO(Brecht): add lookup/LtGadget
+        let is_short = cb.query_cell();
+        let is_long =  cb.query_cell();
+        let is_very_long = cb.query_cell();
+        let is_string = cb.query_cell();
+        
+        circuit!([meta, cb], {
+            require!(vec![
+                FixedTableTag::RLP.expr(), 
+                bytes[0].clone(), 
+                is_string.expr(), 
+                is_short.expr(), 
+                is_very_long.expr()] => @"fixed"
```

Comment:
`is_long` seems to be missing? I think that's actually smart to only do 2 out of 3 in the lookup table because it saves a cell and you can deduce one of the three values e.g. `is_long` is `1 - is_short - is_very_long`, but I don't see that being done so not sure what's up? :)